aiida/orm/computers.py

```diff
@@ -720,6 +731,158 @@ def get_configuration(self, user=None):
 
         return config
 
+    @property
+    def name(self):
+        """Return the computer name.
+
+        .. deprecated:: 1.4.0
+            Will be removed in `v2.0.0`, use the `label` property instead.
+        """
+        warnings.warn('this property is deprecated, use `label` instead', AiidaDeprecationWarning)  # pylint: disable=no-member
+        return self._backend_entity.name
```

Comment:
you might want to use the  `label` property instead (in case we need to change something there)

Reply:
the backend entity doesn't have a `label` property though. The plan was to correct this once all the `name` related code is ripped out just before `v2.0`. Since it is not public facing API we can change that at any time

Reply:
I meant. self.label

Reply:
I see, will do

Reply:
Not sure I was clear - I meant to simply replace this by `return self.label`, since we're repeating ourselves here which might (in principle) lead to divergence between `self.label` and `self.get_label()`.

In any case, since this is just one line and is going to be removed eventually, it's just minor and I'll approve now.